proto/proto.bzl

```diff
@@ -220,6 +220,12 @@ rust_proto_library = rule(
             default = PROTO_COMPILE_DEPS,
         ),
         "_cc_toolchain": attr.label(default = "@bazel_tools//tools/cpp:current_cc_toolchain"),
+        "_rust_tool_wrapper": attr.label(
+            default = "@io_bazel_rules_rust//rust/private/rust_tool_wrapper:rust_tool_wrapper",
+            executable = True,
+            allow_single_file = True,
+            cfg = "host",
```

Comment:
These should be `exec`, not `host` - `host` is for where bazel itself is running, `exec` is for where actions are executed (and throughout)

Reply:
Makes sense, should we change _optional_output_wrapper I believe it needs to run on the exec environment as well.

Reply:
Yes, I believe so :) Thanks!

Reply:
Also https://docs.bazel.build/versions/master/skylark/rules.html says:
"Tools that are executed as part of the build (e.g., compilers, code generators) should be built for the host configuration. In this case, specify cfg="host" in the attribute."

Reply:
exec is not documented, is it deprecated? 
https://docs.bazel.build/versions/master/skylark/lib/attr.html

Reply:
The other way around actually - `exec` was introduced relatively recently, because people were running into issues with its lack :) See also the discussion in https://github.com/bazelbuild/rules_rust/issues/327

Reply:
Awesome!!! Thanks a lot for the info! I'll migrate the other already existing executable attributes in later PRs!